ASC-1040 Refactor Test Case with Steps

ryan-rs opened the pull request:
Update the TestVerifyMBUInstall test case to use the new test case with steps
workflow provided by ASC tooling.

johnduarte commented on molecule/default/tests/test_verify_mbu_installed.py:
```diff
@@ -10,15 +16,41 @@
+# ======================================================================================================================
+# Helpers
+# ======================================================================================================================
+def run_on_container(command, container, run_on_host):
```
Should this helper be removed and the one from https://github.com/rcbops/pytest-rpc/blob/master/pytest_rpc/helpers.py#L510-L527 ?

johnduarte commented on molecule/default/tests/test_verify_mbu_installed.py:
```diff
@@ -10,15 +16,41 @@
+# ======================================================================================================================
+@pytest.mark.test_id('406be222-cbee-11e8-ae3a-0025227c8120')
+@pytest.mark.jira('asc-262')
+@pytest.mark.test_case_with_steps()
```
With the exception of this addition, are the other changes in this file relevant to using steps or should they be broken out into a separate commit?

ryan-rs replied:
@johnduarte Split PR into multiple commits to address your feedback.

ryan-rs committed:
ASC-1040 Enable Test Case with Steps Linting

Update requirements and constraints to include RC release of ASC tooling
to enable test cases with steps.

ryan-rs committed:
ASC-1040 Refactor Test Case with Steps

Update the TestVerifyMBUInstall test case to use the new test case with steps
workflow provided by ASC tooling. Remove 'conftest.py' as it contains the old
way of creating test cases with steps.

ryan-rs committed:
ASC-1040 Update Styling of Test File

Use comment blocks to indicate relevant sections of the test file.
